query-service/api.raml

```diff
@@ -2,293 +2,400 @@
-              body:
-                text/plain:
-                  type: any
-                  example: dlrow olleh
-            403:
-              description: |
-                Forbidden
-                
-                The job has not yet completed and thus cannot yet be queried for
-                files.
-              body:
-                application/json:
-                  type: error.ForbiddenError
-            404:
-              description: |
-                Not Found
-                
-                Either the target `{job-id}` was not found or no file with the
-                name `{file-name}` exists.
-              body:
-                application/json:
-                  type: error.NotFoundError
-            500:
-              description: |
-                Internal Server Error
-                
-                An unexpected exception was thrown while attempting to process
-                the request.
-              body:
-                application/json:
-                  type: error.ServerError
+            required: false
+        body:
+          text/plain:
+            type: string
+            displayName: Job Query
+            description: Raw query text
+      401:
+        description: Unauthorized
+        body:
+          application/json:
+            type: error.UnauthorizedError
+      404:
+        description: Not Found
+        body:
+          application/json:
+            type: error.NotFoundError
+      500:
+        description: Server Error
+        body:
+          application/json:
+            type: error.ServerError
 
-  /statuses:
-    post:
-      description: |
-        Lookup the statuses for the list of job IDs posted.
-        
-        The response will be a mapping whose keys are the posted job IDs and
-        whose values are the status of the job represented by the associated ID.
-        
-        If a job ID in the input list does not appear to be a real/valid job ID,
-        the ID will be omitted from the response object.
-        
-        Each status value will be a valid JobStatus enum item, meaning they will
-        each be one of:
-        
-        * "queued"
-        * "in-progress"
-        * "complete"
-        * "failed"
-        * "expired"
-      body:
-        application/json:
-          type: array
-          items:
+# ---------------------------------------------------------------------------- #
+#                                                                              #
+#   Job Result Retrieval Endpoint                                              #
+#                                                                              #
+# ---------------------------------------------------------------------------- #
+/jobs/{job-id}/result:
+  displayName: Job Result
+  uriParameters:
+    job-id:
+      type: string
+      pattern: ^[\dA-Fa-f]{32}$
+      minLength: 32
+      maxLength: 32
+  get:
+    displayName: Download Job Result
```

Comment:
Don't you need another download query parameter here to influence the Content-Disposition header declared below?

Reply:
Updated.